serde/pom.xml

```diff
@@ -48,12 +51,40 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
+      <scope>test</scope>
```

Comment:
NOTE : This was changed to scope test to avoid a slf4j implementation jar being a transitive dependency for consuming projects

Reply:
Good catch, it wasn't my intention at all! Thanks! :+1: